_includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-merchant-portal-core-feature.md

```diff
@@ -73,6 +74,27 @@ class GuiTableConfig extends SprykerGuiTableConfig
 }
 ```
 
+**src/Pyz/Zed/AclMerchantPortal/AclMerchantPortalConfig.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\AclMerchantPortal;
+
+use Spryker\Zed\AclMerchantPortal\AclMerchantPortalConfig as SprykerAclMerchantPortalConfig;
+
+class AclMerchantPortalConfig extends SprykerAclMerchantPortalConfig
+{
+    /**
+     * @return bool
+     */
+    public function isMerchantToMerchantUserConjunctionByUsernameEnabled(): bool
+    {
+        return true;
+    }
+}
+```
+
```

Comment:
* On line 150 we have an issue with formatting. 
* The following plugin(MerchantPortalAclEntityMetadataConfigExpanderPlugin) has to be substituted by the \Spryker\Zed\AclMerchantPortal\Communication\Plugin\AclEntity\MerchantPortalConfigurationAclEntityMetadataConfigExpanderPlugin.
* MerchantAclMerchantPostCreatePlugin => \Spryker\Zed\AclMerchantPortal\Communication\Plugin\Merchant\MerchantAclEntitiesMerchantPostCreatePlugin
* MerchantAclMerchantUserPostCreatePlugin => MerchantUserAclEntitiesMerchantUserPostCreatePlugin